torch/_decomp/__init__.py

```diff
@@ -298,6 +298,7 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
             aten.floor_divide,
             aten.frac,
             aten.frac_,
+            aten.frexp,
```

Comment:
`core_aten_decompositions` should not decompose into prims. We should just lower `aten.frexp` directly in inductor.